src/main/java/org/reactome/release/qa/common/QACheckerHelper.java

```diff
@@ -54,14 +54,7 @@ public static boolean isChimeric(GKInstance rle) throws Exception {
 	
 	@SuppressWarnings("unchecked")
 	public static GKInstance getHuman(MySQLAdaptor dba) throws Exception {
-	    Collection<GKInstance> c = dba.fetchInstanceByAttribute(ReactomeJavaConstants.Species,
-	            ReactomeJavaConstants._displayName,
-	            "=", 
-	            "Homo sapiens");
-	    if (c == null || c.size() == 0)
-	        throw new IllegalStateException("Cannot find species Homo sapiens in the database, " + 
-	                                        dba.getDBName() + "@" + dba.getDBHost());
-	    return c.iterator().next();
+	    return dba.fetchInstance(HUMAN_SPECIES_INSTANCE_DBID);
```

Comment:
I would recommend a private static GKInstance variable in this class, call it `humanInst`. When `getHuman` is called, check to see if `humanInst` is null, and if it is, then do the `dba.fetchInstance` and then _set_ the private static `humanInst` to whatever comes back from the database. Once it's been set, you can just return the same object every time - no need to keep sending the same query to the database every time a "human" is needed.